src/content/docs/dropins/checkout/checkout-functions.mdx

```diff
@@ -1,9 +1,563 @@
+
+A promise that resolves to an `EmailAvailability` object indicating whether the email is available.
+
+```ts
+type EmailAvailability = boolean;
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { isEmailAvailable } from '@/checkout/api/isEmailAvailable';
+
+isEmailAvailable('test@example.com');
+```
+
+## placeOrder
+
+Handles the process of placing an order in the checkout system using the [`placeOrder`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/place-order/) Adobe Commerce GraphQL API mutation.
+
+### Returns
+
+The transformed order data fetched from the API.
+
+```ts
+transformOrder
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { placeOrder } from '@/checkout/api/placeOrder';
+
+placeOrder('4L7E1H0muABuggqWxbzBCBWoeG0rNxDh');
+```
+
+## redirect
+
+A function that redirects the user to a URL. It takes a single parameter (`url`) of type `string`, which represents the URL to redirect to.
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { redirect } from '@/checkout/api/redirect';
+
+redirect('https://example.com');
+```
+
+## resetCheckout
+
+Resets the checkout process to its initial state by clearing the cart data.
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { state } from '@/checkout/lib/state';
+
+resetCheckout();
+```
+
+## resetCustomer
+
+Clears the current customer information and resets the customer state to its inital value in the checkout process.
+
+### Usage
+
+See the following example for usage details:
+
+```typescript
+import { state } from '@/checkout/lib/state';
+
+resetCustomer();
+```
+
+## setBillingAddress
+
+Sets the billing address for a specific cart using the [`setBillingAddressOnCart`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/set-billing-address/) Adobe Commerce GraphQL mutation.
+
+It takes the following parameters:
+
+- `address`: An object that contains the billing address details.
+- `customer_address_id`: The ID of the customer address.
+- `same_as_shipping`: A boolean value that indicates whether the billing address is the same as the shipping address.
+- `use_for_shipping`: A boolean value that indicates whether the billing address should be used as the shipping address.
+
+### Returns
+
+The function returns a promise that resolves to the transformed cart data fetched from the API.
+
+```ts
+transformCart
+```
+
+### Usage
+
+To set the billing address as the same as the shipping address:
+
+```ts
+import { setBillingAddress } from '@/checkout/api/setBillingAddress';
+
+setBillingAddress({
+  same_as_shipping: true,
+});
+```
+
+To set an specific billing address:
+
+```ts
+import { setBillingAddress } from '@/checkout/api/setBillingAddress';
+setBillingAddress({
+  address: {
+    firstname: 'John',
+    lastname: 'Doe',
+    street: ['123 Main St', 'Apt 1'],
+    city: 'New York',
+    state: 'NY',
```

Comment:
`state` is not part of the schema. It should be `region` or maybe a `region` object